content/pt/docs/5.configuration-glossary/11.configuration-generate.md

```diff
@@ -164,53 +164,53 @@ export default {
 }
 ```
 
-The path to the fallback HTML file. It should be set as the error page, so that also unknown routes are rendered via Nuxt. If unset or set to a falsy value, the name of the fallback HTML file will be `200.html`. If set to `true`, the filename will be `404.html`. If you provide a string as a value, it will be used instead.
+O caminho para o ficheiro HTML alternativo. Ele deve ser definido como a página de erro, assim também as rotas desconhecidas serão renderizadas através do Nuxt. Se for definida ou não para um valor de falsidade, o nome de um ficheiro HTML alternativo será `200.html`. Se for definido para `true`, o nome de ficheiro será `404.html`. Se você fornecer uma sequência de caracteres como um valor, ela será usada no lugar.
 
 ```{}[nuxt.config.js]
 fallback: false;
 ```
 
-If working with statically generated pages then it is recommended to use a `404.html` for error pages and for those covered by [excludes](/docs/configuration-glossary/#exclude) (the files that you do not want generated as static pages).
+Se estiver trabalhando com páginas estaticamente geradas então é recomendado usar um ficheiro `404.html` para páginas de erros e para aqueles cobertos pelo [excludes](/docs/configuration-glossary/#exclude) (os ficheiros que você não quer que sejam gerados como páginas estáticas).
```

Comment:
Does that `#exclude` work? Isn't the title translated?

Reply:
True, it should be broken.
I will test that when enabling `pt-pt` in the i18n config 👌 Portuguese is not enabled so it can't be verified on the preview.
In the meantime we should pay attention to those anchors to that to minimize the broken links.

Reply:
@smarroufin how do you think I should to evaluate it?
@ChristopheCVB do you have any suggestions about how to solve it?

Because even in the original English version of the documentation
this link is written the same way with nothing missing in it.

Is it an internal link to the section #exclude above?

Reply:
Yes it is a link to the above section. The link is broken in english as well, I'll fix it for the other languages.
You should write it this way for PT: `/docs/configuration-glossary/configuration-generate/#a-propriedade-exclude` (the path was incorrect initially, and you translated the title in the page so the anchor has changed)

Reply:
Actually you can simply put `[excludes](#exclude)` since this is the same page

Reply:
@smarroufin `[excludes](#a-propriedade-exclude)` you mean?

Reply:
Sorry, you're right, I lost myself replacing it everywhere 🤦

Reply:
Very thanks to all of you, I will make the corrections right now.